nixos/modules/hardware/dell-command-configure.nix

```diff
@@ -0,0 +1,66 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.dellCommandConfigure;
+  
+  configFile = pkgs.writeTextFile {
+    name = "omreg.cfg";
+    text = ''
+hapi.omilcore.version=9.3.0
```

Comment:
would be nicer with some more indentation

Reply:
Good point, fixed it up.